api/v1/kubtest.yaml

```diff
@@ -500,9 +500,29 @@ components:
           example: 
             users: "3"
             prefix: "some-"
-        metadata: 
+        content: 
           type: string
           description: script content as string (content depends from executor)
+        input-type:
+          type: string
+          description: >
+            script content type can be: 
+            - direct content - created from file, 
+            - git repo directory checkout in case when test is some kind of project or have more than one file, 
+            - or file uri
+          enum:
+            - content
```

Comment:
should we have a binary-content? (which would be an encoded string)

Reply:
i think we could easily refactor when we'll get some executor which could execute such input